python/cudf/cudf/core/multiindex.py

```diff
@@ -535,68 +543,6 @@ def __repr__(self):
         data_output = "\n".join(lines)
         return output_prefix + data_output
 
-    @classmethod
```

Comment:
Do we completely no longer need the `from_arrow` method? Trying to understand how this will now work with MultiIndex after the refactor?